MAUI/StepProgressBar/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: About .NET MAUI Step ProgressBar control | Syncfusion
+description: Learn about the overview of Syncfusion .NET MAUI Step ProgressBar (SfStepProgressBar) control, its basic features, and step progressBar functionalities.
+platform: maui
+control: SfStepProgressBar
+documentation: ug
+---
+
+# Overview of .NET MAUI Step ProgressBar (SfStepProgressBar)
+
+Syncfusion .NET MAUI Step ProgressBar (SfStepProgressBar) control allows you to displays progress through multiple steps in a process, such as order tracking or a user registration form. It's customizable, with options for step shape, content, progress bar color, and more.
+
+## Key features
+
+* **Orientation options**: The Step ProgressBar allows you to change the orientation to horizontal or vertical direction.
```

Comment:
horizontal or vertical- Highlight keywords

Reply:
Highlighted the keywords